tests/large_test/Makefile

```diff
@@ -40,7 +40,7 @@ $(KIICLIENT):
 	cp $(KIICLIENT) ./
 
 $(KIISDK):
-	$(MAKE) -C ../../kii $(KIISDKPARAM) "CFLAGS+=-DKII_JSON_FIXED_TOKEN_NUM=256"
+	$(MAKE) -C ../../kii $(KIISDKPARAM) "CFLAGS+=-DKII_JSON_FIXED_TOKEN_NUM=256 -DKII_PUSH_KEEP_ALIVE_INTERVAL_SECONDS=30"
```

Comment:
I think this is out of scope. And I think, compiler flag better mentioned on README.